plone/scale/storage.py

```diff
@@ -14,6 +14,15 @@
 # See AnnotationStorage._keep_scale_seconds instead.
 KEEP_SCALE_MILLIS = 24 * 60 * 60 * 1000
 
+try:
+    long
+except NameError:
+    # Python 3 has no long, only int.
```

Comment:
Well, for 1.4.x its imo not necessary to fight with Python 3. For master thats different.